### PR TITLE
Changes based on new job class

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@shtripat @brainfunked @r0h4n @nnDarshan @anivargi @nthomas-redhat @anmolbabu please review this

File: tendrl/node_agent/flows/import_cluster/__init__.py [hunk 1]
```diff
@@ -21,25 +22,23 @@ def run(self):
         integration_id = self.parameters['TendrlContext.integration_id']
         tendrl_ns.tendrl_context.integration_id = integration_id
         tendrl_ns.tendrl_context.save()
-        node_list = self.parameters['Node[]']
+        node_list = self.parameters['Node[]'].split(", ")
```
Comment (shtripat) [inline tendrl/node_agent/flows/import_cluster/__init__.py]: Node[] is command separated list now? Have you published a new format of job structure to API team? I feel a good comment on this PR would help and tag @anivargi for this.

File: tendrl/node_agent/objects/definition/master.py [hunk 2]
```diff
@@ -117,6 +117,36 @@
       list: nodes/$Node_context.node_id/Services
       help: "Service"
       value: nodes/$Node_context.node_id/Services
+    Job:
+      attrs:
```
Comment (shtripat) [inline tendrl/node_agent/objects/definition/master.py]: I dont see `parent` in this attrs list

File: tendrl/node_agent/flows/import_cluster/__init__.py (hunk 1 above)
Comment (brainfunked) [inline tendrl/node_agent/flows/import_cluster/__init__.py]: The jobs are created as json structures. I don't see why we have comma separated lists. Lists should be arrays.

**Comment by GowthamShanmugam** [comment]
@anivargi @brainfunked @shtripat @r0h4n please review this

**Pull request merged**